interface/batchcom/emailnotification.php

```diff
@@ -1,9 +1,34 @@
 <?php
+/**
+ * emailnotification script.
+ *
+ * LICENSE: This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://opensource.org/licenses/gpl-license.php>;.
+ *
+ * @package OpenEMR
+ * @author  Brady Miller <brady.g.miller@gmail.com>
+ * @author  Jason 'Toolbox' Oettinger <jason@oettinger.email>
+ * @link    http://www.open-emr.org
+ * @copyright Copyright (c) 2017 Brady Miller <brady.g.miller@gmail.com>
+ * @copyright Copyright (c) 2017 Jason 'Toolbox' Oettinger <jason@oettinger.email>
+ * @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ * @todo    KNOWN SQL INJECTION VECTOR
+ */
+
 //INCLUDES, DO ANY ACTIONS, THEN GET OUR DATA
 include_once("../globals.php");
 include_once("$srcdir/registry.inc");
 include_once("../../library/acl.inc");
 include_once("batchcom.inc.php");
+use OpenEMR\Core\Header;
 
```

Comment:
change line 38 to:
``````
echo "<p>" . xlt('You are not authorized for this.') . "</p>";